public/css/list/item-list.less

```diff
@@ -169,4 +165,9 @@
     max-height: 7.5em; /* 5 lines */
     position: relative;
   }
+
+  .caption,
+  .caption > * {
```

Comment:
Why is this also applied to the plugin output but only in detailed mode? Doesn't it need to be applied always anyway? If not, please remove.

Reply:
I'll change the `*` from line 170 to `.plugin-output`.

`.line-clamp` itself should always be applied, but in detailed it's 4 lines, otherwise 2.